raiden/tests/integration/rpc/test_assumptions.py

```diff
@@ -9,6 +9,16 @@
 from raiden.network.rpc.client import JSONRPCClient
 from raiden.utils.solc import compile_files_cwd
 
+try:
```

Comment:
This is related to how `eth-tester` works - it calls the evm directly on executing contract's `call()`, `estimateGas()` or `transfer()` . If `require` is triggered, it'll raise an exception.
Two exceptions that I am trying to import are raised only when using `eth-tester` as a backend. And because tester is a dev dependency, this makes (and fakes) the exceptions to make python happy in case the library isn't available.